README.md

```diff
@@ -92,21 +80,19 @@ sfdx plugins
 
 <!-- commands -->
```

Comment:
@mshanemc , I assume this stuff in the README is generated form the command help, right?  if so, can you update the code with my comments below so that the --help output is correct?  Thx.